WooCommerce/Classes/ViewRelated/Dashboard/Blaze/BlazeCampaignDashboardView.swift

```diff
@@ -143,7 +149,7 @@ private extension BlazeCampaignDashboardView {
                 Image(uiImage: .blaze)
                     .resizable()
                     .frame(width: Layout.logoSize * scale, height: Layout.logoSize * scale)
-                Text("Blaze")
+                Text(Localization.title)
```

Comment:
Nit: since we changed the name of the card here, we should also change its name in the Customize screen for consistency:

<img src="https://github.com/woocommerce/woocommerce-ios/assets/5533851/e3a6925c-58b0-458f-871d-426bf4873ae2" width=320 />